components/common-data-items.js

```diff
@@ -509,6 +517,14 @@ export function OntologyTermDataItems({ item, isA, children }) {
           </DataItemValue>
         </>
       )}
+      {(item.description || item.definition) && (
```

Comment:
I think term needs to show definition if there is one. if not then show description if there is one. Description may not be as accurate as definition.

Reply:
Oh really? I guess Cricket has it reversed then.

Reply:
I am positive definition is more accurate.